### ISSUE TITLE
[new-dashboard] Personalize dropdown is rendered with a top margin

### ISSUE THREAD

**Issue by bdpiparva** [gocd/gocd#4529]
##### Issue Type
- Bug Report
##### Summary
Personalize dropdown is rendered with a top margin.

![screen shot 2018-03-16 at 9 09 55 am](https://user-images.githubusercontent.com/7871209/37502847-20b1570a-28fb-11e8-814f-c754d4bf1858.png)

##### Environment
```
Go Version: 18.3.0 (6276-ef43aae094874a43081686113d3867a68713ac11) 
```

**Comment by yankurk** [comment]
Fixed as a part of #4522

**Comment by rajiesh** [comment]
verified on 18.3.0 (6530-f9dc89e397d1b134f4d124e6381f8764e38e0a1d)
